src/error.rs

```diff
@@ -96,6 +116,51 @@ pub struct Transport {
     source: Option<Box<dyn error::Error + Send + Sync + 'static>>,
 }
 
+impl Transport {
+    /// The type of error that happened while processing the request.
+    pub fn kind(&self) -> ErrorKind {
+        self.kind
+    }
+
+    /// Higher level error details, if there are any.
+    ///
+    /// ```
+    /// use ureq::ErrorKind;
+    /// use std::error::Error;
+    /// use url::ParseError;
+    ///
+    /// let result = ureq::get("broken/url").call();
+    /// let error = result.unwrap_err().into_transport().unwrap();
+    ///
+    /// // the display trait is a combo of the underlying classifications
+    /// assert_eq!(error.to_string(),
+    ///     "Bad URL: failed to parse URL: RelativeUrlWithoutBase: relative URL without a base");
+    ///
+    /// // classification
+    /// assert_eq!(error.kind(), ErrorKind::InvalidUrl);
+    /// assert_eq!(error.kind().to_string(), "Bad URL");
+
```

Comment:
```suggestion
///
```

This blank line was missing its comment intro.